widget/embedded/src/store/slices/data.ts

```diff
@@ -27,6 +27,7 @@ type TokenOptions = {
 export type FetchStatus = 'loading' | 'success' | 'failed';
 export interface DataSlice {
   _blockchainsMapByName: Map<string, BlockchainMeta>;
+  _tokensMap: Map<string, Token>;
   _tokens: Token[];
```

Comment:
I don't think we need `_tokens` anymore, Wherever we need it, we can create it from _tokensMap.

You can get an idea from `blockchains` function, like this:

```
    const blockchainsFromState = Array.from(
      blockchainsMapByName?.values() || []
    );
```

Reply:
fixed